tgui/packages/tgui/interfaces/TTV.tsx

```diff
@@ -0,0 +1,107 @@
+/**
+ * @file
+ * @copyright 2024
+ * @author Romayne (https://github.com/MeggalBozale)
+ * @license ISC
+ */
+
+import { useBackend } from '../backend';
+import { Button, LabeledList, RoundGauge, Section, Stack } from '../components';
+import { Window } from '../layouts';
+import { toTitleCase } from '../../common/string';
+import { formatPressure } from '../format';
+interface AirVendorParams {
```

Comment:
This should probably be `TransferValveParams` or something. Also generally convention is to put the type definitions in a separate `type.ts` file.

Reply:
:+1: name is a mistake

Reply:
moved interface to subfolder 'TTV' with index.tsx and type.ts